test/accessGroups.js

```diff
@@ -9,9 +10,43 @@ chai.use(chaiHttp);
 
 var app;
 var u;
+var u2;
 var userIdentity;
+var userIdentity2;
 
-before(function() {
+var testUndefinedInstrumentGroup = {
```

Comment:
This is my mistake but it seems that I can't edit your PR. can you change this to `testUndefinedSharedWith`

Reply:
thanks! I had indeed to open a new PR because I couldn't modify yours from the fork, and that unfortunately dis-enables you